Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist

```diff
@@ -150,24 +150,119 @@ A profile can consist of payloads with different version numbers. For example, c
+							<key>pfm_description</key>
+							<string>Example: smb://filer.your.domain.tld/share. Note: %USERNAME% will be replaced with the current user's login name.</string>
+							<key>pfm_name</key>
+							<string>networkShare</string>
+							<key>pfm_require</key>
+							<string>always</string>
+							<key>pfm_format</key>
+							<string>^(smb|afp|https)://.*$</string>
+							<key>pfm_type</key>
+							<string>string</string>
+							<key>pfm_value_placeholder</key>
+							<string>smb://filer.yourdomain.tld/share</string>
+						</dict>
+						<dict>
+							<key>pfm_default</key>
+							<string>krb</string>
+							<key>pfm_title</key>
+							<string>Authentication type</string>
+							<key>pfm_description</key>
+							<string>Authentication type for the share, it can be either through Kerberos (krb) or using a username/password (auth). Default: Kerberos.</string>
+							<key>pfm_name</key>
+							<string>authType</string>
+							<key>pfm_require</key>
+							<string>always</string>
+							<key>pfm_range_list</key>
+							<array>
+								<string>krb</string>
+								<string>password</string>
+							</array>
+							<key>pfm_range_list_titles</key>
+							<array>
+								<string>Kerberos</string>
+								<string>Password</string>
+							</array>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_title</key>
+							<string>Username (Optional)</string>
+							<key>pfm_description</key>
+							<string>Optional: Predefine a username for authentication using username/password</string>
+							<key>pfm_name</key>
+							<string>username</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_title</key>
+							<string>Mount point name (Optional)</string>
+							<key>pfm_description</key>
+							<string>Optional: Change the mount point name for the network share. Leave blank for the default value (recommended)</string>
+							<key>pfm_name</key>
+							<string>mountPoint</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+					</array>
+					<key>pfm_type</key>
+					<string>dictionary</string>
+				</dict>
+			</array>
+			<key>pfm_type</key>
+			<string>array</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>⚠️ Deprecated with version 3. Value will be removed in a future version of the Network Share Mounter.</string>
 			<key>pfm_name</key>
 			<string>networkShares</string>
 			<key>pfm_subkeys</key>
 			<array>
 				<dict>
 					<key>pfm_format</key>
-					<string>^smb:\/\/\S*\/.+$</string>
+					<string>^smb://\S*/.+$</string>
 					<key>pfm_type</key>
 					<string>string</string>
 					<key>pfm_value_placeholder</key>
 					<string>smb://filer.your.domain/share</string>
 				</dict>
 			</array>
 			<key>pfm_title</key>
-			<string>List of Network Shares to Mount</string>
+			<string>Network shares (Deprecated)</string>
```

Comment:
Instead of changing the title to reflect deprecation, could you add the `pfm_app_deprecated` tag to the subkey?